cmd/api/src/test/integration/harnesses.go

```diff
@@ -2127,6 +2127,653 @@ func (s *ESC6aHarnessTemplate2) Setup(c *GraphTestContext) {
+	s.User6 = graphTestContext.NewActiveDirectoryUser("User6", domainSid)
+	s.User7 = graphTestContext.NewActiveDirectoryUser("User7", domainSid)
+	graphTestContext.NewRelationship(s.CertTemplate2, s.EnterpriseCA, ad.PublishedTo)
+	graphTestContext.NewRelationship(s.RootCA, s.Domain, ad.RootCAFor)
+	graphTestContext.NewRelationship(s.EnterpriseCA, s.RootCA, ad.IssuedSignedBy)
+	graphTestContext.NewRelationship(s.NTAuthStore, s.Domain, ad.NTAuthStoreFor)
+	graphTestContext.NewRelationship(s.EnterpriseCA, s.NTAuthStore, ad.TrustedForNTAuth)
+	graphTestContext.NewRelationship(s.EnterpriseCA, s.DC, ad.CanAbuseUPNCertMapping)
+	graphTestContext.NewRelationship(s.DC, s.Domain, ad.DCFor)
+	graphTestContext.NewRelationship(s.User3, s.CertTemplate3, ad.Enroll)
+	graphTestContext.NewRelationship(s.CertTemplate3, s.EnterpriseCA, ad.PublishedTo)
+	graphTestContext.NewRelationship(s.CertTemplate4, s.EnterpriseCA, ad.PublishedTo)
+	graphTestContext.NewRelationship(s.User4, s.CertTemplate4, ad.Enroll)
+	graphTestContext.NewRelationship(s.User2, s.CertTemplate2, ad.Enroll)
+	graphTestContext.NewRelationship(s.Group0, s.EnterpriseCA, ad.Enroll)
+	graphTestContext.NewRelationship(s.User2, s.Group0, ad.MemberOf)
+	graphTestContext.NewRelationship(s.User3, s.Group0, ad.MemberOf)
+	graphTestContext.NewRelationship(s.User4, s.Group0, ad.MemberOf)
+	graphTestContext.NewRelationship(s.CertTemplate5, s.EnterpriseCA, ad.PublishedTo)
+	graphTestContext.NewRelationship(s.User5, s.CertTemplate5, ad.Enroll)
+	graphTestContext.NewRelationship(s.User5, s.Group0, ad.MemberOf)
+	graphTestContext.NewRelationship(s.User6, s.Group0, ad.MemberOf)
+	graphTestContext.NewRelationship(s.User6, s.CertTemplate6, ad.Enroll)
+	graphTestContext.NewRelationship(s.CertTemplate6, s.EnterpriseCA, ad.PublishedTo)
+	graphTestContext.NewRelationship(s.CertTemplate1, s.EnterpriseCA, ad.PublishedTo)
+	graphTestContext.NewRelationship(s.User1, s.CertTemplate1, ad.Enroll)
+	graphTestContext.NewRelationship(s.User1, s.Group0, ad.MemberOf)
+	graphTestContext.NewRelationship(s.CertTemplate7, s.EnterpriseCA, ad.PublishedTo)
+	graphTestContext.NewRelationship(s.User7, s.CertTemplate7, ad.Enroll)
+	graphTestContext.NewRelationship(s.User7, s.Group0, ad.MemberOf)
+	graphTestContext.NewRelationship(s.Group7, s.User7, ad.GenericAll)
+	graphTestContext.NewRelationship(s.Group6, s.User6, ad.GenericAll)
+	graphTestContext.NewRelationship(s.Group5, s.User5, ad.GenericAll)
+	graphTestContext.NewRelationship(s.Group4, s.User4, ad.GenericAll)
+	graphTestContext.NewRelationship(s.Group3, s.User3, ad.GenericAll)
+	graphTestContext.NewRelationship(s.Group2, s.User2, ad.GenericAll)
+	graphTestContext.NewRelationship(s.Group1, s.User1, ad.GenericAll)
+	graphTestContext.NewRelationship(s.Group3, s.Domain, ad.ADCSESC10a)
+	graphTestContext.NewRelationship(s.Group2, s.Domain, ad.ADCSESC10a)
+	graphTestContext.NewRelationship(s.Group1, s.Domain, ad.ADCSESC10a)
```

Comment:
I believe these are the rels being asserted on in the test so we should not create them in the harness and instead let post processing create them from the other relationships that are setup for 10a to be viable.